lib/utils/scroll_controller_extension.dart

```diff
@@ -0,0 +1,7 @@
+import 'package:flutter/material.dart';
+
+extension ScrollControllerExtension on ScrollController {
+  bool get shouldLoadMore {
+    return position.extentAfter < 500;
```

Comment:
Is it support for web version?

Reply:
500 is distance to end, so same with Web and Mobile

Reply:
should be a constant

Reply:
done